src/EditorFeatures/Test2/Rename/InlineRenameTests.vb

```diff
@@ -209,6 +214,27 @@ class Program
             End Using
         End Function
 
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.Rename)>
+        <WorkItem(21657, "https://github.com/dotnet/roslyn/issues/21657")>
+        Public Async Function RenameAttributeCSharp() As Task
+            Using workspace = CreateWorkspaceWithWaiter(
+                    <Workspace>
+                        <Project Language="C#" CommonReferences="true">
+                            <Document>
+using System;
+
+class [|$$ustom|]Attribute : Attribute 
+{
+}
+                            </Document>
+                        </Project>
+                    </Workspace>)
+
+                Await VerifyRenameOptionChangedSessionCommit(workspace, "ustom", "C", isRenameAttribute:=True)
```

Comment:
Ok, next commit I'll rollback this file. 

When user interaction with rename Attribute is different from other, because when rename because is rename a variable he rename entire word and when the user rename Attribute he rename only word before Attribute

Reply:
@CyrusNajmabadi Is this a problem with the `VerifyRenameOptionChangedSessionCommit` method? Since it assumes the edit span and replacement span will be the same before and after commit?

Would passing in a `commitTextSuffix` which defaults to empty string and is concatenated onto the `checkReplacementText` after commit be a suitable compromise?

Reply:
@CyrusNajmabadi Is this a problem with the `VerifyRenameOptionChangedSessionCommit` method? Since it assumes the edit span and replacement span will be the same before and after commit?

Would passing in a `commitTextSuffix` which defaults to empty string and is concatenated onto the `checkReplacementText` after commit be a suitable compromise?

Reply:
> Would passing in a commitTextSuffix which defaults to empty string and is concatenated onto the checkReplacementText after commit be a suitable compromise?

I still don't like it :) 

As i mentioned above, tests should only declare what the input is, what operation the user performed, and the expected output.  Specifying additional data (for example, a suffix to append) just means the test is now trying to do something differnet, and is trying to hardcode in knowledge about a impl level detail.

Reply:
Still rubs me the wrong way that `VerifyRenameOptionChangedSessionCommit` is making expectations of the pre-commit span.

I agree that the attribute rename case is covered by the integration tests and this file can be rolled back.